roles/docker/tasks/main.yml

```diff
@@ -109,6 +109,7 @@
 
 - include: lvm.yml
   when: docker_lvm_backed|bool
+  and: provider!=bare-metal
```

Comment:
I'm not sure this is valid Ansible syntax. You might want:
```yaml
    when: docker_lvm_backend|bool and provider != bare-metal
```
It also looks like the conditional has a typo right now (`backed->backend`), so we might want to fix that too!